From e678bb9cebde4b6b1be36fe75b24ddd2672d835c Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Mon, 5 Sep 2022 06:54:42 +0000
Subject: [PATCH] s32_common.mk: Print error message for debugging

There might be an race issue:
| make: *** [plat/nxp/s32/s32_common.mk:279: /path/to/atf-s32g/2.5-r0/build/s32g2xxaevb/release/dummy_fip.s32] Error 127

Print error message to help debug it.

Upstream-Status: Pending

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 plat/nxp/s32/s32cc/s32_common.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/nxp/s32/s32cc/s32_common.mk b/plat/nxp/s32/s32cc/s32_common.mk
index d359070f4..1b0af2c94 100644
--- a/plat/nxp/s32/s32cc/s32_common.mk
+++ b/plat/nxp/s32/s32cc/s32_common.mk
@@ -367,7 +367,7 @@ endif
 
 ${DUMMY_FIP_S32}: ${DUMMY_FIP}
 	${ECHO} "  MKIMAGE $@"
-	${Q}$(call run_mkimage, ${BL2_BASE}, ${BL2_BASE}, ${MKIMAGE_CFG}, $<, $@) 2> /dev/null
+	${Q}$(call run_mkimage, ${BL2_BASE}, ${BL2_BASE}, ${MKIMAGE_CFG}, $<, $@)
 
 ${IVT_LOCATION_FILE}: ${DUMMY_FIP_S32}
 	${ECHO} "  MKIMAGE $@"
-- 
2.35.5

